source/gui/settingsDialogs.py

```diff
@@ -3488,12 +3488,12 @@ def onDeleteFingerprints(self, evt: wx.CommandEvent) -> None:
 			pgettext(
 				"remote",
 				# Translators: This message is presented when the user tries to delete all stored trusted fingerprints.
-				"This will cause NVDA to forget all previously trusted Remote servers. "
-				"When connecting to a previously trusted unauthorized Remote server, you will again be asked whether to trust its certificate.\n\n"
+				"This will cause NVDA to forget all previously trusted Remote Access servers. "
+				"When connecting to a previously trusted unrecognised Remote Access server, you will again be asked whether to trust its certificate.\n\n"
```

Comment:
I feel like there's no need to repeat Remote Access

```suggestion
				"When connecting to a previously trusted unrecognised server, you will again be asked whether to trust its certificate.\n\n"
```